packages/core/lib/commands/migrate/meta.js

```diff
@@ -51,7 +57,7 @@ module.exports = {
       "                                " + // spacing to align with previous line
       "[--compile-all] [--compile-none] [--verbose-rpc] [--interactive]\n" +
       "                                " + // spacing to align with previous line
-      "[--skip-dry-run] [--describe-json] [--dry-run]",
+      "[--skip-dry-run] [--describe-json] [--dry-run] [--save <boolean>]",
```

Comment:
So since we want to leave this undocumented you should probably leave it out here.